### DISCUSSION TITLE
Light theme by default

### DISCUSSION THREAD

**Discussion by alexgleason** [vuejs/vitepress#3957] [Q&A] [answered]
Hello, I'm having trouble making Vitepress use the **light** theme by default, but still allow the user to change the theme.

Here is what I want:

- Device preferences are ignored
- The theme is light by default
- The user can toggle dark mode

Is this possible?

**Comment by brc-dd** [comment, marked as answer]
try this

```ts
import { defineConfig } from 'vitepress'

export default defineConfig({
  appearance: {
    // @ts-expect-error not fully supported yet
    initialValue: 'light'
  }
})
```

**Reply by alexgleason** [reply]
This works, thank you!

**Reply by brc-dd** [reply]
Let us know if you encounter any problem with that. If there isn't any weird edge case we can add it to types.

**Reply by alexgleason** [reply]
I only confused myself at first because the toggle value was already set in localStorage. After clearing that, it works as expected.

**Reply by brc-dd** [reply]
Ah, yeah that's expected. For new users (you can try in incognito maybe) it should be light regardless of their device preference.

**Reply by SimonEast** [reply]
Thanks for the tip. 

It's a bit confusing how `appearance: 'dark'` makes dark the default, but `appearance: { initialValue: 'light' }` makes light the default. But anyway, if this has been working well for others, perhaps we could add it to the docs?
